v1/account/all.md

```diff
@@ -39,6 +39,22 @@ curl -X GET \
 | account_name | string    | AccountName is the owner name of the account. |
 | major_type   | MajorType | MajorType is the type of account.             |
 
+##### Enums {#enums-CheckAccountResponse}
+
+###### MajorType
+
+MajorType describes the type of the account.
+
+| Value            | Description                                 |
+|------------------|---------------------------------------------|
+| UnknownMajorType |                                             |
+| Checking         | Checking account.                           |
+| Saving           | Saving account.                             |
+| TimeDeposit      | TimeDeposit for a time deposit account.     |
+| CommercialLoan   | CommercialLoan for a business loan account. |
+| MortgageLoan     | MortgageLoan for a home loan account.       |
+| ConsumerLoan     | ConsumerLoan for a consumer loan account.   |
```

Comment:
Do we need the description with ending with `period` `.`, can we remove it inside the table?

Reply:
I don't want to do it programatically, we should delete the dots from comments if we don't want them, rather than do some trimming in gunk

(I agree with you, just saying where to do it)

Reply:
It should not be gunk's work to enforce styling. The trailing . should be removed by users themselves.

Reply:
I thought it was gunk added it, ignore this comment.